plugins/modules/aci_interface_policy_link_level.py

```diff
@@ -231,7 +231,7 @@ def main():
         link_level_policy=dict(type="str", aliases=["name"]),
         description=dict(type="str", aliases=["descr"]),
         auto_negotiation=dict(type="bool", default="true"),
-        speed=dict(type="str", default="inherit", choices=["100M", "1G", "10G", "25G", "40G", "50G", "100G", "200G", "400G", "inherit"]),
+        speed=dict(type="str", default="inherit", choices=["100M", "1G", "10G", "25G", "40G", "50G", "100G", "200G", "400G", "auto", "inherit"]),
```

Comment:
please also update documentation above to add the choice of auto

Reply:
could you add testcase that covers speed? I see the current test cases are not having asserts for all properties, would be great if you could have a look and increase the coverage.

Reply:
Hi @akinross , that can i fix. I take a look tonight and update this PR with the documentation and test cases.

Reply:
Thanks! Please re-request review when you are done because then I receive a notification.